website/docs/docs/on-premises/installation/migrating-from-pact-broker.md

```diff
@@ -0,0 +1,39 @@
+---
+id: migrating
+title: Migrating from Pact Broker
+---
+
+# Migrating from Pact Broker
+
+If you have previously used the open source Pact Broker with a PostgreSQL database it is possible to migrate that data to Pactflow, keeping everything and making it available in Pactflow. The migration process uses the existing Pact Broker database, database migrations are then performed to add additional tables and data used for Pactflow features.
```

Comment:
I'm a grammar pedant. This is called comma splicing, and I never realised what it was until I had a friend who is an even worse grammar pedant than me call me out out on it multiple times. "The migration process uses the existing Pact Broker database, database migrations are then performed to add additional tables and data used for Pactflow features." should either be two sentences, or have an "and" after the comma.